e2e/test-component/scenarios/embedding-sdk/mantine-styles-tests.cy.spec.tsx

```diff
@@ -0,0 +1,142 @@
+// To apply cypress-level mantine styles
+import "@mantine/core/styles.css";
+import "@mantine/dates/styles.css";
+
+// Mics behavior when an app adds its own styles
+// We need to do it before importing the SDK
+import "e2e/support/helpers/embedding-sdk-helpers/host-app-styles.css";
```

Comment:
This is important to mimic the real app behavior when it:
- imports its own mantine styles
- imports its own custom styles that may override Mantine styles (with the same specificity, so only the order between Mantine and custom styles matters)
- imports SDK

We have the same setup for Shoppy